.github/workflows/android-perf.yml

```diff
@@ -0,0 +1,198 @@
+name: android-perf
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  # TODO: Remove 'pull_request' before merge
+  pull_request:
+  push:
+    tags:
+      - ciflow/perf-android
+  # Note: GitHub has an upper limit of 10 inputs
+  workflow_dispatch:
+    inputs:
+      models:
+        description: Models to be benchmarked
+        required: false
+        type: string
+        default: 'stories110M'
+      devices:
+        description: Target devices to run benchmark
+        required: false
+        type: string
+        default: false
+      delegates:
+        description: Backend delegates
+        required: false
+        type: string
+        default: xnnpack
+      threadpool:
+        description: Run with threadpool?
+        required: false
+        type: boolean
+        default: false
+      benchmark_configs:
+        description: The list of configs used the benchmark
+        required: false
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}-${{ github.event_name == 'schedule' }}
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  set-models:
+    runs-on: linux.2xlarge
+    outputs:
+      models: ${{ steps.set-models.outputs.models }}
+    steps:
+      - name: Set models
+        id: set-models
+        run: |
+          MODELS="${{ github.event.inputs.models }}"
+          if [[ -z "$MODELS" ]]; then
+            echo "No models provided and failed to retrieve the default values."
+            exit 1
+          fi
```

Comment:
@huydhn Is this the correct way to retrieve the workflow inputs? I assume it will retrieve either the provided value or default value, however, I always end up getting `null` on reading `${{ github.event.inputs.models }}`.

Reply:
Ah, sorry, it's just `inputs.*`, i.e. `${{ inputs.models }}`

Reply:
Let me give a try